tools/nni_cmd/config_schema.py

```diff
@@ -1,8 +1,9 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
-import os
 import json
+import logging
+import os
```

Comment:
what is the order of imported object? `os` should be after `netifaces` in character order?